[2023-12-07 17:00:11,369] ERROR log_exception() L1449 Exception on /avaliacoes [GET] - call_trace=C:\Python311\Lib\site-packages\flask\app.py L1449
Traceback (most recent call last):
  File "C:\Python311\Lib\site-packages\flask\app.py", line 2073, in wsgi_app
    response = self.full_dispatch_request()
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Python311\Lib\site-packages\flask\app.py", line 1520, in full_dispatch_request
    return self.finalize_request(rv)
           ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Python311\Lib\site-packages\flask\app.py", line 1539, in finalize_request
    response = self.make_response(rv)
               ^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Python311\Lib\site-packages\flask\app.py", line 1714, in make_response
    rv = jsonify(rv)
         ^^^^^^^^^^^
  File "C:\Python311\Lib\site-packages\flask\json\__init__.py", line 302, in jsonify
    f"{dumps(data, indent=indent, separators=separators)}\n",
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Python311\Lib\site-packages\flask\json\__init__.py", line 132, in dumps
    return _json.dumps(obj, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Python311\Lib\json\__init__.py", line 238, in dumps
    **kw).encode(obj)
          ^^^^^^^^^^^
  File "C:\Python311\Lib\json\encoder.py", line 200, in encode
    chunks = self.iterencode(o, _one_shot=True)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Python311\Lib\json\encoder.py", line 258, in iterencode
    return _iterencode(o, 0)
           ^^^^^^^^^^^^^^^^^
  File "C:\Python311\Lib\site-packages\flask\json\__init__.py", line 51, in default
    return super().default(o)
           ^^^^^^^^^^^^^^^^^^
  File "C:\Python311\Lib\json\encoder.py", line 180, in default
    raise TypeError(f'Object of type {o.__class__.__name__} '
TypeError: Object of type bytes is not JSON serializable
[2023-12-07 17:00:11,473] INFO _log() L224  127.0.0.1 - - [07/Dec/2023 17:00:11] "[35m[1mGET /avaliacoes HTTP/1.1[0m" 500 - - call_trace=C:\Python311\Lib\site-packages\werkzeug\_internal.py L224 
[2023-12-07 17:01:08,742] INFO _log() L224  127.0.0.1 - - [07/Dec/2023 17:01:08] "[33mGET /pacientes HTTP/1.1[0m" 404 - - call_trace=C:\Python311\Lib\site-packages\werkzeug\_internal.py L224 
[2023-12-07 17:01:10,958] INFO _log() L224  127.0.0.1 - - [07/Dec/2023 17:01:10] "[33mGET /pacientes HTTP/1.1[0m" 404 - - call_trace=C:\Python311\Lib\site-packages\werkzeug\_internal.py L224 
[2023-12-07 17:08:18,771] INFO _log() L224   * Running on all addresses (0.0.0.0)
   WARNING: This is a development server. Do not use it in a production deployment.
 * Running on http://127.0.0.1:5000
 * Running on http://192.168.0.179:5000 (Press CTRL+C to quit) - call_trace=C:\Python311\Lib\site-packages\werkzeug\_internal.py L224 
[2023-12-07 17:08:24,811] INFO _log() L224  127.0.0.1 - - [07/Dec/2023 17:08:24] "[32mGET / HTTP/1.1[0m" 302 - - call_trace=C:\Python311\Lib\site-packages\werkzeug\_internal.py L224 
[2023-12-07 17:08:24,891] INFO _log() L224  127.0.0.1 - - [07/Dec/2023 17:08:24] "GET /openapi/ HTTP/1.1" 200 - - call_trace=C:\Python311\Lib\site-packages\werkzeug\_internal.py L224 
[2023-12-07 17:08:25,266] INFO _log() L224  127.0.0.1 - - [07/Dec/2023 17:08:25] "[36mGET /openapi/static/images/swagger.svg HTTP/1.1[0m" 304 - - call_trace=C:\Python311\Lib\site-packages\werkzeug\_internal.py L224 
[2023-12-07 17:08:25,320] INFO _log() L224  127.0.0.1 - - [07/Dec/2023 17:08:25] "[36mGET /openapi/static/images/redoc.svg HTTP/1.1[0m" 304 - - call_trace=C:\Python311\Lib\site-packages\werkzeug\_internal.py L224 
[2023-12-07 17:08:25,561] INFO _log() L224  127.0.0.1 - - [07/Dec/2023 17:08:25] "[36mGET /openapi/static/images/rapidoc.svg HTTP/1.1[0m" 304 - - call_trace=C:\Python311\Lib\site-packages\werkzeug\_internal.py L224 
[2023-12-07 17:08:29,501] INFO _log() L224  127.0.0.1 - - [07/Dec/2023 17:08:29] "GET /openapi/swagger HTTP/1.1" 200 - - call_trace=C:\Python311\Lib\site-packages\werkzeug\_internal.py L224 
[2023-12-07 17:08:29,557] INFO _log() L224  127.0.0.1 - - [07/Dec/2023 17:08:29] "[36mGET /openapi/static/css/swagger-ui.css HTTP/1.1[0m" 304 - - call_trace=C:\Python311\Lib\site-packages\werkzeug\_internal.py L224 
[2023-12-07 17:08:29,601] INFO _log() L224  127.0.0.1 - - [07/Dec/2023 17:08:29] "[36mGET /openapi/static/js/swagger-ui-bundle.js HTTP/1.1[0m" 304 - - call_trace=C:\Python311\Lib\site-packages\werkzeug\_internal.py L224 
[2023-12-07 17:08:29,625] INFO _log() L224  127.0.0.1 - - [07/Dec/2023 17:08:29] "[36mGET /openapi/static/js/swagger-ui-standalone-preset.js HTTP/1.1[0m" 304 - - call_trace=C:\Python311\Lib\site-packages\werkzeug\_internal.py L224 
[2023-12-07 17:08:30,344] INFO _log() L224  127.0.0.1 - - [07/Dec/2023 17:08:30] "GET /openapi/openapi.json HTTP/1.1" 200 - - call_trace=C:\Python311\Lib\site-packages\werkzeug\_internal.py L224 
[2023-12-07 17:08:35,784] INFO _log() L224  127.0.0.1 - - [07/Dec/2023 17:08:35] "POST /avaliacao HTTP/1.1" 200 - - call_trace=C:\Python311\Lib\site-packages\werkzeug\_internal.py L224 
[2023-12-07 17:09:28,950] INFO _log() L224  127.0.0.1 - - [07/Dec/2023 17:09:28] "[31m[1mPOST /avaliacao HTTP/1.1[0m" 422 - - call_trace=C:\Python311\Lib\site-packages\werkzeug\_internal.py L224 
[2023-12-07 17:09:38,740] INFO _log() L224  127.0.0.1 - - [07/Dec/2023 17:09:38] "POST /avaliacao HTTP/1.1" 200 - - call_trace=C:\Python311\Lib\site-packages\werkzeug\_internal.py L224 
[2023-12-11 16:58:26,027] INFO _log() L224   * Running on all addresses (0.0.0.0)
   WARNING: This is a development server. Do not use it in a production deployment.
 * Running on http://127.0.0.1:5000
 * Running on http://192.168.0.179:5000 (Press CTRL+C to quit) - call_trace=C:\Python311\Lib\site-packages\werkzeug\_internal.py L224 
[2023-12-11 16:58:29,757] INFO _log() L224  127.0.0.1 - - [11/Dec/2023 16:58:29] "[32mGET / HTTP/1.1[0m" 302 - - call_trace=C:\Python311\Lib\site-packages\werkzeug\_internal.py L224 
[2023-12-11 16:58:30,084] INFO _log() L224  127.0.0.1 - - [11/Dec/2023 16:58:30] "GET /openapi/ HTTP/1.1" 200 - - call_trace=C:\Python311\Lib\site-packages\werkzeug\_internal.py L224 
[2023-12-11 16:58:30,464] INFO _log() L224  127.0.0.1 - - [11/Dec/2023 16:58:30] "[36mGET /openapi/static/images/rapidoc.svg HTTP/1.1[0m" 304 - - call_trace=C:\Python311\Lib\site-packages\werkzeug\_internal.py L224 
[2023-12-11 16:58:30,541] INFO _log() L224  127.0.0.1 - - [11/Dec/2023 16:58:30] "[36mGET /openapi/static/images/swagger.svg HTTP/1.1[0m" 304 - - call_trace=C:\Python311\Lib\site-packages\werkzeug\_internal.py L224 
[2023-12-11 16:58:30,573] INFO _log() L224  127.0.0.1 - - [11/Dec/2023 16:58:30] "[36mGET /openapi/static/images/redoc.svg HTTP/1.1[0m" 304 - - call_trace=C:\Python311\Lib\site-packages\werkzeug\_internal.py L224 
[2023-12-11 16:58:31,143] INFO _log() L224  127.0.0.1 - - [11/Dec/2023 16:58:31] "GET /avaliacoes HTTP/1.1" 200 - - call_trace=C:\Python311\Lib\site-packages\werkzeug\_internal.py L224 
[2023-12-11 16:58:31,340] INFO _log() L224  127.0.0.1 - - [11/Dec/2023 16:58:31] "GET /openapi/swagger HTTP/1.1" 200 - - call_trace=C:\Python311\Lib\site-packages\werkzeug\_internal.py L224 
[2023-12-11 16:58:31,499] INFO _log() L224  127.0.0.1 - - [11/Dec/2023 16:58:31] "[36mGET /openapi/static/css/swagger-ui.css HTTP/1.1[0m" 304 - - call_trace=C:\Python311\Lib\site-packages\werkzeug\_internal.py L224 
[2023-12-11 16:58:31,633] INFO _log() L224  127.0.0.1 - - [11/Dec/2023 16:58:31] "[36mGET /openapi/static/js/swagger-ui-bundle.js HTTP/1.1[0m" 304 - - call_trace=C:\Python311\Lib\site-packages\werkzeug\_internal.py L224 
[2023-12-11 16:58:31,956] INFO _log() L224  127.0.0.1 - - [11/Dec/2023 16:58:31] "[36mGET /openapi/static/js/swagger-ui-standalone-preset.js HTTP/1.1[0m" 304 - - call_trace=C:\Python311\Lib\site-packages\werkzeug\_internal.py L224 
[2023-12-11 16:58:32,508] INFO _log() L224  127.0.0.1 - - [11/Dec/2023 16:58:32] "GET /openapi/openapi.json HTTP/1.1" 200 - - call_trace=C:\Python311\Lib\site-packages\werkzeug\_internal.py L224 
[2023-12-11 16:58:32,625] INFO _log() L224  127.0.0.1 - - [11/Dec/2023 16:58:32] "GET /avaliacoes HTTP/1.1" 200 - - call_trace=C:\Python311\Lib\site-packages\werkzeug\_internal.py L224 
[2023-12-11 16:58:45,016] INFO _log() L224  127.0.0.1 - - [11/Dec/2023 16:58:45] "POST /avaliacao HTTP/1.1" 200 - - call_trace=C:\Python311\Lib\site-packages\werkzeug\_internal.py L224 
[2023-12-11 16:58:45,500] INFO _log() L224  127.0.0.1 - - [11/Dec/2023 16:58:45] "GET /avaliacoes HTTP/1.1" 200 - - call_trace=C:\Python311\Lib\site-packages\werkzeug\_internal.py L224 
[2023-12-11 16:58:46,125] INFO _log() L224  127.0.0.1 - - [11/Dec/2023 16:58:46] "GET /avaliacoes HTTP/1.1" 200 - - call_trace=C:\Python311\Lib\site-packages\werkzeug\_internal.py L224 
[2023-12-11 16:59:21,452] INFO _log() L224  127.0.0.1 - - [11/Dec/2023 16:59:21] "GET /avaliacoes HTTP/1.1" 200 - - call_trace=C:\Python311\Lib\site-packages\werkzeug\_internal.py L224 
[2023-12-11 16:59:21,671] INFO _log() L224  127.0.0.1 - - [11/Dec/2023 16:59:21] "GET /avaliacoes HTTP/1.1" 200 - - call_trace=C:\Python311\Lib\site-packages\werkzeug\_internal.py L224 
[2023-12-11 16:59:33,253] INFO _log() L224  127.0.0.1 - - [11/Dec/2023 16:59:33] "POST /avaliacao HTTP/1.1" 200 - - call_trace=C:\Python311\Lib\site-packages\werkzeug\_internal.py L224 
[2023-12-11 16:59:35,475] INFO _log() L224  127.0.0.1 - - [11/Dec/2023 16:59:35] "GET /avaliacoes HTTP/1.1" 200 - - call_trace=C:\Python311\Lib\site-packages\werkzeug\_internal.py L224 
[2023-12-11 17:10:24,608] INFO _log() L224   * Running on all addresses (0.0.0.0)
   WARNING: This is a development server. Do not use it in a production deployment.
 * Running on http://127.0.0.1:5000
 * Running on http://192.168.0.179:5000 (Press CTRL+C to quit) - call_trace=C:\Python311\Lib\site-packages\werkzeug\_internal.py L224 
